iina/uk.lproj/MainWindowController.strings

```diff
@@ -34,5 +34,5 @@
 "eCe-SS-Wlj.title" = "-32x";
 
 /* Class = "NSTextFieldCell"; title = "This video is playing in picture in picture"; ObjectID = "gdE-b5-kod"; */
-"gdE-b5-kod.title" = "Це відео відтворюється в режимі «Кадр у кадрі»";
+"gdE-b5-kod.title" = "Це відео відтворюється в режимі «Картинка в картинці»";
```

Comment:
Wrong. The red one is used in official Ukrainian localisation of macOS apps (eg., Safari web browser).
<img width="292" alt="Screenshot 2019-05-06 16 38 04" src="https://user-images.githubusercontent.com/36358174/57229233-3ca09f00-701e-11e9-9104-12819044c9e9.png">